src/components/Filters/components/ConnectedFilterControl/ConnectedFilterControl.scss

```diff
@@ -57,8 +57,8 @@ $stacking-order: (
 }
 
 .newDesignLanguage {
-  .CenterContainer + .RightContainer {
-    margin-left: spacing(extra-tight);
+  .CenterContainer {
+    margin-right: spacing(extra-tight);
```

Comment:
Looks like this should be using spacing(tight) or spacing() to equal the space between filters and the save button

Reply:
I think that's better as well!